SWLOR.Web/ClientApp/package-lock.json

```diff
@@ -1,8 +1,17621 @@
 {
```

Comment:
Not sure what caused this file to be changed but can you undo the changes to it?

Reply:
Done. Copied the file from `ffo-rewrite` and pasted it over the file in this branch. Hopefully that should do it.